src/galaxy/app.py

```diff
@@ -1050,23 +1087,101 @@ def run_ogr2ogr_cmd(cmd,dump_temp_file_path,binding_file_dir):
             cmd,
             stdout=subprocess.PIPE,
             stderr=subprocess.PIPE,
-            shell=True
+            shell=True,
+            preexec_fn=os.setsid
```

Comment:
shell = true for creating an intermediate shell process for request , and preexec_fn will asign the id for the process so that if anything happens or goes over limit we can kill that shell with process id